process-engine/bpmn-studio
# 🏗Make Login Dependent from Solution

**Changes:**

1. Make it possible to login for a single solution and not for the entire application.
2. Delete the Config Panel.

PR: #PullRequest

## How can others test the changes?

1. Start an IdentityServer on port 5000.
2. Start an IdentityServer on port 5001.
3. Start a runtime with the authority set to port 5001.
4. Start BPMN-Studio. (Remember: The internal PE has it authority set to port localhost:5000 by default)
5. Login via the new button right from the solution name.
6. See in the network tab of the developer tools that only the requests on the logged in solution have a legit bearer token.
7. Logout via the same button.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
